[Dashboards] Recreate the data-exploration tutorial dashboard with the Dashboards API (#6337)
elastic/docs-content: explore-analyze/kibana-data-exploration-learning-tutorial.md

6. **Log volume over time, per host**: the line chart from the [Add a line chart of log volume over time](#add-a-line-chart-of-log-volume-over-time) sub-step. The reference line at value `80` is a separate `reference_lines` layer in the same panel.
7. **Events by response code**: the {{esql}} bar chart saved to the dashboard from Discover in [Step 2](#explore-data-in-discover). ES|QL chart layers reference query result columns directly in `x` and `y` (for example, `"x": { "column": "response" }`), instead of the `operation`-based form used by data view layers.
8. **Requests by file extension**: bar chart from the [Add a bar chart of requests by file extension](#add-a-bar-chart-of-requests-by-file-extension) sub-step. The `includes` filter with `as_regex: true` and value `.+` mirrors the regex applied during the inline-edit step.
9. **Traffic distribution by operating system**: pie chart from the [Expand your dashboard](#expand-your-dashboard) sub-step. Pie panels use `config.type: "pie"` with a `metrics` array and a `group_by` array.

> 10. **Requests by geography**: treemap from the same sub-step. Treemaps use the same `metrics` + `group_by` shape as pies.

Reviewer: Optional, I but I find it more readable to say 
```suggestion
10. **Requests by geography**: treemap from the [Expand your dashboard](#expand-your-dashboard) sub-step. Treemaps use the same `metrics` + `group_by` shape as pies.
```
It's repetitive but I think it's a bit more clear.

For more details on sharing options, access control, and managing dashboard ownership, refer to [Sharing dashboards](dashboards/sharing.md).

## (Optional) Recreate the dashboard with the API [recreate-dashboard-api]

> 

Author: ```suggestion
## Recreate the dashboard with the API [recreate-dashboard-api]
```

```{applies_to}
stack: preview 9.4+
serverless: preview
```

Everything you built in this tutorial can also be reproduced in a single API call. The [Dashboards API](dashboards/create-dashboards-programmatically.md) accepts a JSON payload that encodes the complete dashboard, including panel types, data sources, layout, and display options, making it straightforward to version-control dashboards or provision consistent environments programmatically.

<!--
  The curl example below is verified end-to-end by
  .github/scripts/verify-dashboards-api-example.py. The Dashboards API is in
  technical preview, so its schema can change between releases. Re-run that
  script (with KIBANA_URL and API_KEY set) whenever you edit the payload, or
  whenever the spec at https://github.com/elastic/dashboards-api-spec is
  updated. Last verified against dashboards-api-spec @ 84120e3 on 2026-05-06.
-->
::::{dropdown} Recreate this dashboard with one API call

> The following `curl` request creates the same dashboard as the one you just built, including the optional panels suggested earlier in the step:

Author: ```suggestion
The following `curl` request creates the same dashboard as the one you built in this tutorial, including the optional panels suggestions.
```
